e2e-tests/tfm/cypress/e2e/journeys/utilisation-reports/add-payment.spec.js

```diff
@@ -68,23 +68,23 @@ context('PDC_RECONCILE users can add a payment to a report', () => {
   });
 
   it('should render the selected fee record details', () => {
-    pages.utilisationReportAddPaymentPage.selectedReportedFeesDetailsTable().contains('11111111');
-    pages.utilisationReportAddPaymentPage.selectedReportedFeesDetailsTable().contains('Exporter 1');
-    pages.utilisationReportAddPaymentPage.selectedReportedFeesDetailsTable().contains('JPY 100');
-    pages.utilisationReportAddPaymentPage.selectedReportedFeesDetailsTable().contains('GBP 50');
+    pages.utilisationReportAddPaymentPage.selectedReportedFeesDetailsTable().contains('11111111').should('exist');
```

Comment:
On reading the difference between these, I found that we might actually be able to do `pages.utilisationReportAddPaymentPage.selectedReportedFeesDetailsTable().should('contain', '11111111')` which I think I like the most?

Reply:
I tried to find that in the assertions sections of the cypress documentation, but couldn't seem to. On a second look, it seems like it is at the bottom of the list of assertions which is why I missed it 🙃